Chat/src/java/pro/nechyporenko/javabegin/chat/entity/Message.java

```diff
@@ -0,0 +1,30 @@
+package java.pro.nechyporenko.javabegin.chat.entity;
+
+public class Message {
+
+    private int id;
+    private String text;
```

Comment:
As we discussed with Alexander, there should be more fields according to db scheme, also should be present table Users and DAO for it. DB dump should be also overridden. Check with Alexander.